### PR TITLE
first_js

### PR DESCRIPTION
**Pull request by BartlomiejWrz**
check this code!!!

### CODE REVIEW

File: exercise.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+for (var i = 0; i <= 100; i++) {
+
```
Comment (lukaszbasaj) [inline exercise.js]: Blank space is unnecessary

File: exercise.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    if (i % 3 === 0 && i % 5 === 0) {
+        console.log("fizzbuzz");
+    }
+
```
Comment (lukaszbasaj) [inline exercise.js]: Blank space is unnecessary

File: exercise.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    else if (i % 3 === 0) {
+        console.log("fizz");
+    }
+
```
Comment (lukaszbasaj) [inline exercise.js]: Blank space is unnecessary

File: exercise.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    else if (i % 5 === 0) {
+        console.log("buzz");
+    }
+
```
Comment (lukaszbasaj) [inline exercise.js]: Blank space is unnecessary

File: exercise.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    else {
+        console.log(i)
+    }
+
```
Comment (lukaszbasaj) [inline exercise.js]: Blank space is unnecessary

File: exercise.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (lukaszbasaj) [inline exercise.js]: Blank space is unnecessary

**Review by lukaszbasaj — CHANGES_REQUESTED** [review]

**Review by lukaszbasaj — APPROVED** [review]

**Review by BartlomiejWrz — COMMENTED** [review]
Ok, you're wright. Alredy improved

**Pull request closed without merging**